src/pages/User/content/SpaceProfile.tsx

```diff
@@ -207,14 +143,47 @@ const renderMachineBuilderXp = (machineBuilderXp: Array<IMAchineBuilderXp>) => (
     <h4>We offer the following services:</h4>
     {machineBuilderXp.map((machineExperience, index) => {
       return (
-        <MachineExperienceTab key={`machineXp-${index}`}>
+        <Box
+          sx={{
+            backgroundColor: 'background',
+            borderColor: 'background',
+            borderRadius: '5px',
+            borderStyle: 'solid',
+            borderWidth: '1px',
+            display: 'inline-block',
+            marginRight: '10px',
+            padding: '10px',
+          }}
+          key={`machineXp-${index}`}
+        >
           {machineExperience}
-        </MachineExperienceTab>
+        </Box>
       )
     })}
   </>
 )
 
+const MobileBadge = ({ children }) => (
+  <Flex
+    sx={{
+      flexDirection: 'column',
+      marginBottom: 0,
+      position: 'relative',
+      width: '100%',
+      '@media only screen and (min-width: 52em)': {
```

Comment:
I really don't like that I'm defining this breakpoint. Any ideas?

Reply:
Thanks for calling this out @benfurber, I think that we should be able to write this in a shorthand manner using:

```suggestion
      height: ['300px', '300px', '500px'],
```
This would mean that:
* The default height is 300px
* At the 0th breakpoint & larger, height is 300px
* At the 1st breakpoint & larger, height is 500px

It took me some time to get used to this convention of the theming systems but I believe this slightly opaque syntax is the only way to work with breakpoints. 
https://theme-ui.com/sx-prop#responsive-values

Reply:
Thanks @thisislawatts, I think I get it now(!) Does it look right to you now?